actions/set_role_voice_channel_perms_MOD.js

```diff
@@ -57,12 +57,8 @@ module.exports = {
 
   async action(cache) {
     const data = cache.actions[cache.index];
-    const storage = parseInt(data.vchannel, 10);
-    const varName = this.evalMessage(data.varName, cache);
-    const channel = await this.getVoiceChannel(storage, varName, cache);
-    const storage2 = parseInt(data.role, 10);
-    const varName2 = this.evalMessage(data.varName2, cache);
-    const role = await this.getRole(storage2, varName2, cache);
+    const channel = await this.getVoiceChannelFromData(data.storage, data.varName, cache);
+    const role = await this.getRoleFromData(data.storage2, data.varName2, cache);
```

Comment:
Use the corresponding HTML elements
The first parameters should be `data.vchannel` and `data.role`